doc/releases/luminous.rst

```diff
@@ -1,3 +1,157 @@
+v12.2.9 Luminous
+================
+
+This is the ninth bug fix release of the Luminous v12.2.x long term stable
+release series. This release contains several bugfixes across all the components
+and we recommend all users upgrade.
+
+Notable Changes
+---------------
+
+[ADD VERBIAGE HERE]
```

Comment:
This could probably be simplified, I think we should warn about increased OSD memory usage with bluestore, e.g.

```suggestion
* OSD

  The bluestore_cache_* options are no longer needed. They are replaced
  by osd_memory_target, defaulting to 4GB. BlueStore will expand
  and contract its cache to attempt to stay within this
  limit. Users upgrading should note this is a higher default
  than the previous bluestore_cache_size of 1GB, so OSDs using
  BlueStore will use more memory by default.
  
  For more details, see `BlueStore docs <http://docs.ceph.com/docs/master/rados/configuration/bluestore-config-ref/#cache-size>_`
```